integration/airflow/openlineage/airflow/extractors/converters.py

```diff
@@ -0,0 +1,16 @@
+from openlineage.client.run import Dataset
+from airflow.lineage.entities import Table
+
+
+def table_to_dataset(table: Table):
+    return Dataset(
+        namespace=f"{table.cluster}",
+        name=f"{table.database}.{table.name}",
+        facets={
+            "tags": table.tags,
```

Comment:
Those should either be standard facets defined in OpenLineage standard, or defined somewhere custom facets.

We can do just `namespace` and `name` for now, and add this stuff later.